src/planner/search/dfs/dfs.cpp

```diff
@@ -1,6 +1,7 @@
 #include "planner/search/dfs/dfs.h"
 
 #include <iostream>
+#include <cmath>
```

Comment:
We can remove this if we don't need.

Reply:
In the building process, the compiler says `std::sqrt` is invalid. That's why I had to add **cmath**.